### PR TITLE
proto: bump to 1.6.x 2f6f381

### PR DESCRIPTION
**Pull request by chhsia0**
Updated Mesos protos to 1.6.x, which adds a new master call for GET_OPERATIONS, and new agent calls for GET_OPERATIONS, GROW_VOLUME and SHRINK_VOLUME. For the latter two, only agent-local volumes are supported.

### CODE REVIEW

File: api/v1/lib/scheduler/scheduler.proto [hunk 1]
```diff
@@ -239,7 +239,8 @@ message Response {
     repeated OperationStatus operation_statuses = 1 [(gogoproto.nullable) = false];
   }
 
-  optional ReconcileOperations reconcile_operations = 1;
+  optional Type type = 1 [(gogoproto.nullable) = false];
```
Comment (jdef) [inline api/v1/lib/scheduler/scheduler.proto]: this tag change is unfortunate but aligns perfectly w/ upstream changes

File: api/v1/lib/master/calls/calls.go [hunk 2]
```diff
@@ -221,6 +228,32 @@ func DestroyVolumes(a mesos.AgentID, v ...mesos.Resource) *master.Call {
 	}
 }
 
+// GrowLocalVolume grows a persistent volume on an agent's ROOT or PATH disks. The request is forwarded asynchronously
+// to the Mesos agent where the persistent volume is located.
+func GrowLocalVolume(a mesos.AgentID, volume mesos.Resource, addition mesos.Resource) *master.Call {
```
Comment (jdef) [inline api/v1/lib/master/calls/calls.go]: nit: it's more idiomatic to say `volume, addition mesos.Resource` (since they've of the same type)

Comment (jdef) [reply api/v1/lib/master/calls/calls.go]: also: mesos.AgentID is optional in the protos, so it should be a pointer here

File: api/v1/lib/master/calls/calls.go [hunk 3]
```diff
@@ -221,6 +228,32 @@ func DestroyVolumes(a mesos.AgentID, v ...mesos.Resource) *master.Call {
...
+			AgentID:  &a,
+			Volume:   volume,
+			Addition: addition,
+		},
+	}
+}
+
+// ShrinkLocalVolume shrinks a persistent volume on an agent's ROOT or PATH disks. The request is forwarded
+// asynchronously to the Mesos agent where the persistent volume is located.
+func ShrinkLocalVolume(a mesos.AgentID, volume mesos.Resource, subtract mesos.Value_Scalar) *master.Call {
```
Comment (jdef) [inline api/v1/lib/master/calls/calls.go]: mesos.AgentID is optional in protos, should be a pointer parameter

**Comment by jdef** [comment]
Thanks, left some feedback. Surprised that schedulerpb_test.go didn't have any changes

File: api/v1/lib/scheduler/scheduler.proto (hunk 1 above)
Comment (chhsia0) [reply api/v1/lib/scheduler/scheduler.proto]: I checked with Gaston and confirmed that this was not used anywhere when introduced, so it should be a "safe" change.

File: api/v1/lib/master/calls/calls.go (hunk 3 above)
Comment (chhsia0) [reply api/v1/lib/master/calls/calls.go]: I'll make it a pointer. Then should I just rename it to `GrowVolume`/`ShrinkVolume`? Currently it supports only local volumes but me may extend it to support external volumes in the future.

**Comment by jdef** [comment]
I'm fine w/ GrowVolume/ShrinkVolume

**Comment by chhsia0** [comment]
@jdef Done. PTAL.

**Pull request merged**